### PR TITLE
[docs] Revamp of feature guides

### LINKED ISSUE #767: Improve documentation
# Documentation Structure

### Getting started (quickstart)
- Installation
- Basic model setup
- Basic forecasting and plotting

### Quick Tutorials (learn): quick rewards, minimal know how to use, give people overview
- Model components
- Plotting, Forecasting, evaluating

### How To Guides (specific): step by step, specific use case, how to solve a specific problem, advanced skills of using NeuralProphet
- The “real” how to guides: all on the same level
    - Migration (p-NP)
    - Data preparation
    - Hyperparameters
    - Validation and testing
- Application examples
    - …

### The science behind (understanding)
- Model architecture
- Presentation
- Link to paper, extracts

### API Reference
- User-facing (forecaster and utils) (emphasize)
- Complete (all)

### Community
- Slack
- Github
- Contributing.md

# Todos

Here goes a draft on how to restructure the documentation. Below there is a draft and a list of further todos.

- [ ] Should we add a welcome page like on github? I'd propose to move the development / contribution instructions to a separate page
- [ ] Should we make it possible to plot only selected components? (e.g. only trend, seasonality, etc). Could be interesting for the tutorials to reduce complexity
- [ ] Show strength and weaknesses of neuralprophet (e.g. in comparison to other models)
- [ ] Overall goals: Amount of explanatory text, code and visualizations should be balanced (approx. 1/3 each)

# Draft

## Getting started

Quickstart: Installation, minimal model, all auto, nice chart in the end, as few lines of code as possible

Incorporate into other sections: Dataset (electricity price might be interesting, very relatable, uses weather as input)

1. Basic model (and plots) - or should plots be a dedicated section? already contains components
2. Trend
3. Seasonality
4. Auto regression
5. Exogenous variables: Lagged and future regressors (atmospheric pressure, humidity, etc) - split in two sections?
6. Events and holidays (do not really connect to weather, might need other dataset?)
7. Uncertainty (TODO)
8. Global model (multiple datasets)
9. Validation (split dataset, forecast, performance metrics)
10. Recap, model overview (summary on components, link to details of neural model) and next steps

## Advanced topics / other topics

1. Model details (neural model, understanding the model, technical architecture)
2. Missing data: Interpolation
3. Normalization - is this a topic for the getting started section?
4. Hyperparameter tuning
5. Difference between multiplicative and additive components
6. Uncertainty: Confomal Prediction and Conformalized Quantile Regression

## New to time series?
Intro: Time series data: link to view resources? Kaggle inspiration

# Inspiration and examples
1. https://pytorch-lightning.readthedocs.io/en/latest/index.html
2. https://docs.djangoproject.com/en/4.1/
3. https://documentation.divio.com/

### PR DESCRIPTION
**Pull request by leoniewgnr**
See here for detailed description of changes: https://github.com/ourownstory/neural_prophet/issues/1297

**Open todos**

- [x] get through all feature guides to check that they don't throw any error
- [x] new structure
- [x] improve global local tutorial
- [x] improve cross-validation
- [x] fix live plot

**For a later commit**
- [ ] extend multiplicative seasonality
- [ ] add regularization tutorial
- [ ] add Multi-step-forecasting

Close #767

### CODE REVIEW

**Review by noxan — APPROVED** [review]
Oh, I love it - amazing work @leoniewgnr and thanks for wrapping up the docs 👏
